### PR TITLE
@feature/86eppzzee/Tabs_for_regQuestion_and_CStudy

### PR DESCRIPTION
**Pull request by acostales08**

### CODE REVIEW

File: apps/web-backoffice-generic/src/pages/manageQuestionaire.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    <>
+      <ControlledTabs
+        value={tabValue}
+        tabsName={tabname}
+        handleChange={handleTabValue}
+      >
+        {tabValue === 0 ? (
+          <RegQuestionaire />
+        ) : (
+          tabValue === 1 && <CaseStudyQustionaire />
```
Comment (JezChristian) [inline apps/web-backoffice-generic/src/pages/manageQuestionaire.tsx:31]: If you only have 2 expected output which is RegQuestionnaire and CaseStudyQuestionnaire

I think you dont need the tabValue === 1 && in your else block?

File: packages/core-library/components/Tabs/tab.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  handleChange,
+  tabsName,
+  children,
+  ...rest
+}) => {
+  return (
+    <Box sx={{ width: "100%" }}>
+      <Box sx={{ borderBottom: 1, borderColor: "divider" }}>
+        <Tabs value={value} onChange={handleChange} {...rest}>
+          {tabsName.map((item: tabsNameProps, index: number) => (
```
Comment (JezChristian) [inline packages/core-library/components/Tabs/tab.tsx:28]: add tabsName.length > 0 && tabsName.map

File: apps/web-backoffice-generic/src/components/blocks/caseStudyQustionaire.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export const CaseStudyQustionaire = () => {
```
Comment (JezChristian) [inline apps/web-backoffice-generic/src/components/blocks/caseStudyQustionaire.tsx:3]: the name is in wrong spelling

File: apps/web-backoffice-generic/src/components/blocks/regQuestionaire.tsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { Paper } from "@mui/material";
+
+export const RegQuestionaire = () => {
```
Comment (JezChristian) [inline apps/web-backoffice-generic/src/components/blocks/regQuestionaire.tsx:4]: double "n" in Questionnaire

File: packages/core-library/components/GenericDrawerLayout/MockMenus.tsx [hunk 5]
```diff
@@ -1,122 +1,103 @@
...
-import { NavigationType } from '../../types/navigation';
-
+import DashboardIcon from "@mui/icons-material/Dashboard";
+import FeedIcon from "@mui/icons-material/Feed";
+import PersonIcon from "@mui/icons-material/Person";
+import QuestionAnswerIcon from "@mui/icons-material/QuestionAnswer";
+import ReportIcon from "@mui/icons-material/Report";
+import SettingsIcon from "@mui/icons-material/Settings";
+import SourceIcon from "@mui/icons-material/Source";
+import { NavigationType } from "../../types/navigation";
```
Comment (JezChristian) [inline packages/core-library/components/GenericDrawerLayout/MockMenus.tsx:9]: Just a question, is it not possible to import them all from a single "@mui/icons-material" ?

File: apps/web-backoffice-generic/src/pages/manageQuestionaire.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
```
Comment (JMSevilla) [inline apps/web-backoffice-generic/src/pages/manageQuestionaire.tsx:1]: Your filename will be consider as route are you sure you will name this as `manageQuestionaire` ? imagine `/hub/manageQuestionnaire` is this weird?

File: apps/web-backoffice-generic/src/pages/manageQuestionaire.tsx [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+    },
+  ];
+
+  const handleTabValue = (event: React.SyntheticEvent, newValue: number) => {
+    setTabValue(newValue);
+  };
+
+  return (
+    <>
+      <ControlledTabs
```
Comment (JMSevilla) [inline apps/web-backoffice-generic/src/pages/manageQuestionaire.tsx:23]: If i'm not mistaken this is copied from my repository, we can improve our tab components not on this way.

**Review by JMSevilla — CHANGES_REQUESTED** [review]

**Review by JMSevilla — APPROVED** [review]

**Review by DropitupQwerty — APPROVED** [review]

**Review by LizterBabael — APPROVED** [review]

**Pull request merged**